styles.css

```diff
@@ -119,28 +118,7 @@ label {
 label img {
 	width: 50px;
 }
-/* https://www.freecodecamp.org/news/css-selectors-cheat-sheet/ */
+
 .lock:checked + label img {
 	content: url(assets/locked.png);
 }
-
-/*Specifc ID*/
-#box1 {
```

Comment:
Nice, We don't need to hardcode the backgrounds for the boxes as we run the generateRandomPalette at the start!